src/alcotest-engine/utils.ml

```diff
@@ -1,5 +1,9 @@
 let ( >> ) f g x = x |> f |> g
 
+module Fun = struct
+  let id x = x
+end
```

Comment:
What's wrong with the one from `Stdlib`? :)

Reply:
It doesn't exist pre-4.08, sadly.